src/dataframerow/dataframerow.jl

```diff
@@ -100,6 +131,20 @@ function isequal_row(df1::AbstractDataFrame, r1::Int, df2::AbstractDataFrame, r2
     return true
 end
 
+function isequivalent_row(df1::AbstractDataFrame, r1::Int, df2::AbstractDataFrame, r2::Int)
+    if df1 === df2
+        if r1 == r2
+            return true
```

Comment:
This can be simplified to
```julia
if df1 === df2
    return r1 == r2
end
```

Reply:
In this case, I can't change this because we don't want to return yet if `r1 != r2`

Reply:
Oh, I see. In that case you can simplify if you'd like to `df1 === df2 && r1 == r2 && return true`.